repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/git/GitHelper.java

```diff
@@ -335,19 +335,28 @@ public void gitAdd(List<String> files, Git git) {
         }
     }
 
-    public void copyDirectory(File sourceDir, File targetDir, String[] excludedFileNames, AbstractStep step) {
+    public void copyDirectory(File sourceDir, File targetDir, String[] excludedFileNames, boolean isToRemove, AbstractStep step) {
```

Comment:
Could you at least add a comment to explain the purpose of the boolean parameter please?

Reply:
Sure. I will do that.

Reply:
This was already solved in another PR.